src/bin/zmlicensectl

```diff
@@ -63,8 +76,17 @@ startLicenseDaemonService()
 	fi
 }
 
+displayHelp()
+{
+	echo "Usage: $0 [option...]"
+	echo
+	echo "   --service start|restart|stop"
```

Comment:
why are we introducing --service parameter here? most of other zm*ctl commands directly accepts start/stop/restart commands, why can't we do same here?

Reply:
We can do same but we may introduce `nalpeiron` daemon in the future, and for that, we also use the same script like `zmlicensectl --nalpeiron start|stop`. @RakeshAMore correct me if I am wrong.